code/modules/projectiles/guns/projectile/launchers.dm

```diff
@@ -83,3 +83,749 @@
 		to_chat(user, "<span class='notice'>You load [num_loaded] spear\s into \the [src].</span>")
 		update_icon()
 		chamber_round()
+		
+#define PINNED (1<<0)
+#define UNPINNED (1<<1)
+#define DEPLOYED (1<<2)
+#define FIRE_READY (1<<3)
+#define SPENT (1<<4)
+
+// M72-LAW(Light Anit-Tank Weapon) -> S72-LRW(Syndicate Light Rocket Weapon)
+/obj/item/gun/projectile/S72LRW
+	name = "S72LRW-HE"
+	desc = "A portable one-shot 66-mm unguided high explosive rocket."
+	icon_state = "SLRW-pinned"
+	item_state = "launcher"
+	w_class = WEIGHT_CLASS_NORMAL
+	throw_range = 1
+	flags = HANDSLOW
+	mag_type = /obj/item/ammo_box/magazine/internal/S72LRW
+	slot_flags = 0
+	fire_sound = 'sound/weapons/grenadelaunch.ogg'
+	recoil = 1
+	var/handle_time = 20
+	var/brace_slow = 3
+	var/weapon_stage = PINNED
+	var/strp_clr = rgb(50, 200, 50) // Stripe color. Green
+	var/obj/item/SLRWpin/pin = new /obj/item/SLRWpin
+	var/deploy_lock = FALSE
```

Comment:
While not required, do you think you could add some comments to the new variables you're adding? They are helpful to outside readers. Same with the some of the other new vars on your projectiles.

Reply:
Let me know if the code needs more comments.